tests/test_array.h

```diff
@@ -0,0 +1,87 @@
+/*************************************************************************/
+/*  test_array.h                                                          */
```

Comment:
Align the end of the comment with the lines above and below:

```suggestion
/*  test_array.h                                                         */
```